.mci.yml

```diff
@@ -100,7 +135,10 @@ buildvariants:
       run_on:
           - ubuntu1404-test
       tasks:
-          - name: compile_and_test
+          - name: compile
```

Comment:
Can you define a variable with this list of tests and use it for both buildvariants?  Would look something like this:

```
tasks: &all_tasks
- compile
- run_examples
- etc

- name: ubuntu-test
  tasks: *all_tasks

- name: ubuntu-debug
  tasks: *all_tasks
```